Docs/Configuration.tex

```diff
@@ -265,35 +265,35 @@ \subsection{Configuration Terms}\label{configuration-terms}
   \texttt{invalid\ value} if missing.
 \item
   \texttt{fatal\ behaviour} --- behaviour leading to boot termination.
-  Implementation must stop the boot process from going any further until
-  next host system boot. It is allowed but not required to perform cold
-  reboot or show any warning message.
+  Implementations must prevent the boot process from proceeding until the
+  next host system boot. It is permitted, but not required, to execute
+  cold reboots or to show warning messages in such cases.
 \item
   \texttt{undefined\ behaviour} --- behaviour not prescribed by this
-  document. Implementation is allowed to take any measures including but
-  not limited to \texttt{fatal\ behaviour}, assuming any states or
-  values, or ignoring, unless these measures negatively affect system
-  security in general.
+  document. Implementations are permitted to take any measures including, but
+  not limited to, \texttt{fatal\ behaviour}, assumptions of any state or
+  value, or disregarding the state or value. This is however subject to such
+  measures not negatively impacting upon system integrity.
 \end{itemize}
 
 \subsection{Configuration Processing}\label{configuration-processing}
 
-\texttt{OC\ config} is guaranteed to be processed at least once if it
-was found. Depending on OpenCore bootstrapping mechanism multiple
-\texttt{OC\ config} files may lead to reading any of them. No
-\texttt{OC\ Config} may be present on disk, in which case all the values
-read follow the rules of \texttt{invalid\ value} and
-\texttt{optional\ value}.
-
-\texttt{OC\ config} has size, nesting, and key amount limitations.
-\texttt{OC\ config} size does not exceed \texttt{16\ MBs}.
-\texttt{OC\ config} has no more than \texttt{8} nesting levels.
-\texttt{OC\ config} has up to \texttt{16384} XML nodes (i.e.~one
+The \texttt{OC\ config} file is guaranteed to be processed at least once if found.
+Depending on OpenCore bootstrapping mechanism, the presence of multiple
+\texttt{OC\ config} files may lead to the reading any of them. It is permissible that
+no \texttt{OC\ Config} file is present on disk. In such cases, all the values
+read shall follow the rules of \texttt{invalid\ values} and
+\texttt{optional\ values}.
+
+The \texttt{OC\ config} file has size, nesting, and key amount limitations.
+The \texttt{OC\ config} file size shall not exceed \texttt{16\ MBs}.
+The \texttt{OC\ config} file shall not have more than \texttt{8} nesting levels.
+The \texttt{OC\ config} file shall have up to \texttt{16384} XML nodes (i.e.~one
```

Comment:
suggest -> "must not"

Reply:
Written some specs in professional life and follow definitions in RFC2119 with a preference for "Shall" over "Must".
See https://www.ietf.org/rfc/rfc2119.txt

Reply:
Well original thumbs up was pre your edit, but happy to leave it post as well!

Reply:
Sticking with "shall" in view of ultimate goal.